### PR TITLE
Feature/standard repo files

### PR DESCRIPTION
**Pull request by jwtlake**
- Added contribution doc
- Looks like we already added a licensing document at some point

Fixes: https://github.com/MRN-Code/penny-collector/issues/14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 # Workflow
 
 1. Create a new branch from the `master` branch of this repo.
 1. Add your tests, features, fixes and documentation.
 1. Run `npm test` to verify that all tests pass.
+1. Review code coverage and ensure adequate coverage
 1. Create a Pull Request on Github.com to merge your branch with the `master` branch.
 1. Assign a code review task to one of your co-workers on Asana.
-1. Pray if its Cory
+1. Pray if it's Cory
```
Comment (swashcap) [inline CONTRIBUTING.md]: Wait a minute…

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Workflow
+
+1. Create a new branch from the `master` branch of this repo.
+1. Add your tests, features, fixes and documentation.
+1. Run `npm test` to verify that all tests pass.
+1. Review code coverage and ensure adequate coverage
+1. Create a Pull Request on Github.com to merge your branch with the `master` branch.
+1. Assign a code review task to one of your co-workers on Asana.
+1. Pray if it's Cory
```
Comment (swashcap) [inline CONTRIBUTING.md]: Please drop this line.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+PENNY-COLLECTOR Contribution Guidelines
+=========
+
+# Workflow
+
+1. Create a new branch from the `master` branch of this repo.
```
Comment (swashcap) [inline CONTRIBUTING.md]: I prefer Markdown lists that contain actually numbered lists. Makes it more readable in its raw format. Care to change?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+1. Pray if it's Cory
+
+For more info see: https://docs.google.com/document/d/1wbiw0hbk1pluzFyWas-yfXQkNFhXT5M1edlnJNdP2vU/edit
+
+# Linting
+
+Please use the provided .eslintrc.js to lint all files. This can be most easily accomplished by running `npm run lint`. We also *strongly* suggest that you incorporate these linting agents into your text-editor for efficiency's sake.
+
+**No linting errors will be tollerated**
+~Robo Lint Cop
```
Comment (swashcap) [inline CONTRIBUTING.md]: I don't mean to be humorless, but this need to be more professional. Please drop the “Robo Lint Cop” line

File: CONTRIBUTING.md (hunk 2 above)
Comment (jwtlake) [inline CONTRIBUTING.md]: removed

File: CONTRIBUTING.md (hunk 4 above)
Comment (jwtlake) [inline CONTRIBUTING.md]: removed

**Pull request merged**